app/src/main/res/xml/preferences.xml

```diff
@@ -1,99 +1,97 @@
 <?xml version="1.0" encoding="utf-8"?>
-<PreferenceScreen
-    xmlns:android="http://schemas.android.com/apk/res/android">
+<PreferenceScreen xmlns:app="http://schemas.android.com/apk/res-auto"
+  xmlns:android="http://schemas.android.com/apk/res/android">
 
-    <fr.free.nrw.commons.ui.LongTitlePreferences.LongTitlePreferenceCategory
-        android:title="@string/preference_category_appearance">
+    <PreferenceCategory
+      android:title="@string/preference_category_appearance">
 
-        <fr.free.nrw.commons.ui.LongTitlePreferences.LongTitleListPreference
```

Comment:
Delete these classes now that they are no longer used and in their place use `app:singleLineTitle="false"`

Reply:
Done!